opensrp-opd/src/main/java/org/smartregister/opd/repository/OpdDetailsRepository.java

```diff
@@ -80,6 +79,11 @@ public boolean saveOrUpdate(@NonNull org.smartregister.opd.pojo.OpdDetails opdDe
         return recordId != -1;
     }
 
+    @Override
+    public boolean save(org.smartregister.opd.pojo.OpdDetails opdDetails) {
+        return saveOrUpdate(opdDetails);
```

Comment:
Make the same changes made in `OpdDiagnosisAndTreatmentFormRepository` here. I think you just need to add an exception in `saveOrUpdate` and copy previous impl. to this method